mythril/analysis/solver.py

```diff
@@ -138,35 +138,28 @@ def get_transaction_sequence(
 def _replace_with_actual_sha(
     concrete_transactions: List[Dict[str, str]], model: z3.Model, code=None
 ):
+    concrete_hashes = keccak_function_manager.get_concrete_hash_data(model)
     for tx in concrete_transactions:
         if hash_matcher not in tx["input"]:
             continue
         if code is not None and code.bytecode in tx["input"]:
             s_index = len(code.bytecode) + 2
         else:
             s_index = 10
-        for i in range(s_index, len(tx["input"])):
+        for i in range(s_index, len(tx["input"]), 64):
```

Comment:
Why did you add back the alignment again?
(with the current abi spec this won't be a problem, but I wonder 😅 )

Reply:
This probably popped up while I was checking where the problem was, will remove this line